xcell/mappers/mapper_DELS.py

```diff
@@ -73,6 +77,7 @@ def _get_angmask(self):
             ipix = hp.ang2pix(nside, cat['RA'], cat['DEC'],
                               lonlat=True)
             self.mskflag = bmask[ipix] > 0.
+            self.mskflag = rotate_mask(self.mskflag, self.rot)
```

Comment:
I don't think this does what you think. `self.mskflag` here is not a sky map, but a flag for each galaxy for whether or not it is unmasked. So don't rotate it!